lib/MusicBrainz/Server/Validation.pm

```diff
@@ -310,6 +311,24 @@ sub is_valid_partial_date
     return 1;
 }
 
+sub is_valid_edit_note
+{
+    my $edit_note = shift;
+
+    # An edit note with only spaces and / or punctuation is useless
+    if ($edit_note =~ /^[\s.,+\-\\\/'"´`@#~()\[\]{}]+$/) {
+        return 0
+    }
+
+    # An edit note with just one ASCII character is useless
+    # A one-character Japanese note (for example) might be useful, so limited to ASCII 
+    if ($edit_note =~ /^[ -~]{1}$/) {
```

Comment:
```suggestion
    if ($edit_note =~ /^[[:ascii:]]$/) {
```
would be slightly clearer

Reply:
I didn't know that was possible! Does seem better, will switch.